### PR TITLE
feature/DOCKER-new-changes-for-jenkins

### PR DESCRIPTION
**Pull request by Andreichenko**
added a new Jenkinsfile

### CODE REVIEW

**Comment by Andreichenko** [comment]
please review it @freideveloper

File: Jenkinsfile [hunk 1]
```diff
@@ -21,6 +21,12 @@ timestamps {
             '''
 
         }
+        stage('Test'){
+          sh '''
+           cat salt-master/Dockerfile
+           cat salt-minion/Dockerfile
+          '''
+        }
```
Comment (freideveloper) [inline Jenkinsfile:29]: OK

**Review by freideveloper — APPROVED** [review]
OK

File: Jenkinsfile (hunk 1 above)
Comment (Andreichenko) [reply Jenkinsfile:29]: thanks

**Pull request merged**